### PR TITLE
Email: Redesign section headers and add links to services in Email page

### PR DESCRIPTION
**Pull request by stephanethomas**
This pull request updates the [`Email` page](https://wordpress.com/email/) for Titan and Google Workspace in order to:

* **Display icons of email providers**: this improves branding as well as consistency with the `Purchases Settings` page
* **Show product names and domain names on two lines**: this is to avoid issues with long names and small viewports
* **Add an `Add New Mailboxes` button for Titan**: that way the user experience is more consistent across email providers
* **Add icons and links to email provider services**: so users can access their mailboxes and other apps easily

Before | After | After (Expanded)
------ | ----- | -----
B | A | screenshot

Note the design was inspired by the [`Connections` page](https://wordpress.com/marketing/connections/). There is some duplicate code between the G Suite and Titan components. Ideally, we should reuse the same components but since this pull request is already long enough I thought this refactoring should take place in another one.

#### Testing instructions

1. Run `git checkout update/email-page` and start your server, or open a [live branch](https://calypso.live/?branch=update/email-page)
2. Log into a WordPress.com account with a Titan or a Google Workspace account
3. Open the [`Email` page](http://calypso.localhost:3000/email)
4. Assert that this page looks like the screenshots above
5. Assert that it still looks fine on small viewports
6. Check that the placeholder displays correctly

### CODE REVIEW

File: client/my-sites/email/email-management/titan-management-nav/index.jsx [hunk 1]
```diff
@@ -78,39 +87,48 @@ class TitanManagementNav extends React.Component {
...
+
+		const summary = domain?.titanMailSubscription?.subscriptionId && (
+			<Button
+				primary
+				compact
+				href={ emailManagementNewTitanAccount( selectedSiteSlug, domain.name, currentRoute ) }
 			>
-				{ translate( 'Add more mailboxes' ) }
-			</VerticalNavItem>
+				{ translate( 'Add New Mailboxes' ) }
```
Comment (a8ci18n) [inline client/my-sites/email/email-management/titan-management-nav/index.jsx:114]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Number of new mailboxes to add' )` *ES Score: 7*

File: client/my-sites/email/email-management/titan-management-nav/index.jsx [hunk 2]
```diff
@@ -78,39 +87,48 @@ class TitanManagementNav extends React.Component {
...
 			<div className="titan-management-nav">
-				<SectionHeader label={ headerLabel } />
+				<FoldableCard
+					className="titan-management-nav__foldable-card"
+					header={ header }
+					summary={ summary }
+					expandedSummary={ summary }
+				>
+					<ul className="titan-management-nav__foldable-card-services">
+						<li><a href="https://wp.titan.email/mail/" target="_blank" rel="noreferrer noopener"><img src={ titanMailIcon } alt={ translate( 'Titan Mail icon' ) } /><strong>{ translate( 'Mail' ) }</strong></a></li>
```
Comment (a8ci18n) [inline client/my-sites/email/email-management/titan-management-nav/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Titan Mail' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/email-page)

Comment (a8ci18n) [inline client/my-sites/email/email-management/titan-management-nav/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **40** times:
`translate( 'mail' )` *ES Score: 9*

File: client/my-sites/email/email-management/titan-management-nav/index.jsx [hunk 3]
```diff
@@ -78,39 +87,63 @@ class TitanManagementNav extends React.Component {
...
+				<FoldableCard
+					className="titan-management-nav__foldable-card"
+					header={ header }
+					summary={ summary }
+					expandedSummary={ summary }
+				>
+					<ul className="titan-management-nav__foldable-card-services">
+						<li>
+							<a href="https://wp.titan.email/mail/" target="_blank" rel="noreferrer noopener">
+								<img src={ titanMailIcon } alt={ translate( 'Titan Mail icon' ) } />
```
Comment (a8ci18n) [inline client/my-sites/email/email-management/titan-management-nav/index.jsx:129]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Titan Mail' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/email-page)

File: client/my-sites/email/email-management/titan-management-nav/index.jsx [hunk 4]
```diff
@@ -78,39 +87,63 @@ class TitanManagementNav extends React.Component {
...
+								<strong>{ translate( 'Mail' ) }</strong>
```
Comment (a8ci18n) [inline client/my-sites/email/email-management/titan-management-nav/index.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **40** times:
`translate( 'mail' )` *ES Score: 9*

File: client/my-sites/email/email-management/titan-management-nav/index.jsx [hunk 5]
```diff
@@ -78,39 +87,79 @@ class TitanManagementNav extends React.Component {
...
+					header={ header }
+					summary={ summary }
+					expandedSummary={ summary }
+				>
+					<ul className="titan-management-nav__foldable-card-services">
+						<li>
+							<a href="https://wp.titan.email/mail/" target="_blank" rel="noreferrer noopener">
+								<img src={ titanMailIcon } alt={ translate( 'Titan Mail icon' ) } />
+								<strong>
+									{ translate( 'Mail', {
```
Comment (a8ci18n) [inline client/my-sites/email/email-management/titan-management-nav/index.jsx:131]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **40** times:
`translate( 'mail' )` *ES Score: 9*

**Review by hambai — COMMENTED** [review]
Very nice Stephane! I have one concern though - I don't see the use case for the FoldableCard. It might as well be just a card that you can see all the time - why the extra click?

**Review by hambai — APPROVED** [review]
I'm also fine merging this as it is if you think the FoldableCard is good or if you want to refactor it in a follow up PR.

**Comment by stephanethomas** [comment]
> I don't see the use case for the FoldableCard. It might as well be just a card that you can see all the time - why the extra click?

Thanks for the review Kamen. I went for this approach in order to keep the overall design of the page as simple as possible. I believe these links are just nice-to-have (we never had any request to add them for G Suite, although it's true that Google services are probably easier to find). They will even be less essential for Titan once we integrate their webmail to WordPress.com itself. Finally, we are in the process of adding more information to this page (such as the number of licenses) so it's probably a good idea to avoid having too many things competing for the user's attention.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/5527521

Thank you @stephanethomas for including a screenshot in the description! This is really helpful for our translators.

**Comment by daledupreez** [comment]
This looks awesome, @stephanethomas! 😍

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/5527521).

**Pull request merged**